hw/top_earlgrey/data/ip/chip_rv_plic_testplan.hjson

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_rv_plic
+  testpoints: [
+    // PLIC integration tests:
+    {
+      name: chip_sw_plic_all_irqs
+      desc: '''Verify all interrupts from all peripherals aggregated at the PLIC.
+
+            The automated SW test enables all interrupts at the PLIC to interrupt the core. It uses
+            the `intr_test` CSR in each peripheral to mock assert an interrupt, looping through all
+            available interrupts in that peripheral. The ISR verifies that the right interrupt
+            occurred. This is used as a catch-all interrupt test for all peripheral integration
+            testing within which functionally asserting an interrupt is hard to achieve or not of
+            high value.
+            '''
+      stage: V2
+      si_stage: SV2
+      lc_states: ["PROD"]
+      bazel: []
+      features: ["RV_PLIC.INTERRUPT_PRIORITY_AND_THRESHOLD", "RV_PLIC.INTERRUPT_ENABLE"]
```

Comment:
Thanks for renaming the priority/threshold feature earlier. I think the renaming needs to be applied here too.

Reply:
updated feature name